examples/open_llama/README.md

```diff
@@ -70,11 +71,21 @@ Inference is done like a normal pytorch model.
 
 The relevant config file is [open_llama_sparsegpt_gpu.json](open_llama_sparsegpt_gpu.json)
 
+Requirements file: [requirements-sparsegpt.txt](requirements-sparsegpt.txt)
+
+### Fine-tune Llama Model using QLoRA
+This workflow fine-tunes LLaMA model using [QLoRA](https://arxiv.org/abs/2305.14314). The output model is still the input transformers model along with a quantization config and
+LoRA adapters that were fine-tuned on the training dataset.
+
+The relevant configfile is [llama_qlora.json](llama_qlora.json). It corresponds to the [guqnaco 7b example in the original qlora implementation](https://github.com/artidoro/qlora/blob/main/scripts/finetune_guanaco_7b.sh).
+
+Requirements file: [requirements-qlora.txt](requirements-qlora.txt)
+
 ## How to run
 ### Pip requirements
-Install the necessary python packages:
+Install the necessary python packages using the corresponding requirements file.
 ```
-python -m pip install -r requirements.txt
+python -m pip install -r <requirements_file>.txt

```

Comment:
If you have some local experiments results, it would be better to paste here to tell: 
1. your device info
2. the model got fine-tuned.
3. some metrics? if we have.

Reply:
Not block this PR.

Reply:
I am still running some experiments to verify the numbers. but I am planning on updating the readme with some numbers once I have them.